fuel-vm/src/gas/default_gas_costs.rs

```diff
@@ -0,0 +1,122 @@
+use super::*;
+
+pub const GIT: &str = "1a660f59b892e7816da01c288c86696c9d7d7a36";
+pub fn default_gas_costs() -> GasCostsValues {
+    GasCostsValues {
+        add: 1,
+        addi: 1,
+        aloc: 1,
+        and: 1,
+        andi: 1,
+        bal: 155,
+        bhei: 1,
+        bhsh: 1,
+        burn: 35,
+        cb: 2,
+        cfei: 1,
+        cfsi: 1,
+        croo: 163,
+        div: 1,
+        divi: 1,
+        ecr: 1601,
+        eq: 1,
+        exp: 1,
+        expi: 1,
+        flag: 1,
+        gm: 1,
+        gt: 1,
+        gtf: 1,
+        ji: 1,
+        jmp: 1,
+        jne: 1,
+        jnei: 1,
+        jnzi: 1,
+        k256: 18,
+        lb: 1,
+        log: 40,
+        lt: 1,
+        lw: 1,
+        mcpi: 3,
+        mint: 34,
+        mlog: 1,
+        mod_op: 1,
+        modi: 1,
+        move_op: 1,
+        movi: 1,
+        mroo: 2,
+        mul: 1,
+        muli: 1,
+        noop: 1,
+        not: 1,
+        or: 1,
+        ori: 1,
+        ret: 1,
+        rvrt: 1,
+        s256: 5,
```

Comment:
since sha256 takes a slice of memory with arbitrary length, wouldn't it also have a dependent cost?